server/views/pages/securityUpcoming.html

```diff
@@ -0,0 +1,83 @@
+{% extends "../partials/layout.html" %}
+
+{% from "button/macro.njk" import govukButton %}
+
+{% from "../macros/breadCrumb.html" import breadCrumb %}
+
+{% block beforeContent %}
+
+  {{ breadCrumb("Categorisation home", breadCrumbList) }}
+
+{% endblock %}
+
+{% block content %}
+
+<div >
+  <h1 class="govuk-heading-l">Categorisation referrals</h1>
+</div>
+
+<div class="govuk-tabs" data-module="govuk-tabs">
+
+  <ul class="govuk-tabs__list">
+    <li class="govuk-tabs__list-item">
+      <a class="govuk-tabs__tab" id="todo-tab" href="/securityHome">
+        To do
+      </a>
+    </li>
+    <li class="govuk-tabs__list-item">
+      <a class="govuk-tabs__tab" id="done-tab" href="/securityDone">
+        Done
+      </a>
+    </li>
+    <li class="govuk-tabs__list-item govuk-tabs__list-item--selected">
+      <a class="govuk-tabs__tab" id="upcoming-tab">
+        Upcoming
+      </a>
+    </li>
+  </ul>
+
+  <section class="govuk-tabs__panel">
+
+  {% if offenders | length > 0 %}
+  <table class="govuk-table tablesorter sortcolumns" id="offenderTable" >
+    <thead class="govuk-table__head">
+
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header" scope="col">Name</th>
+      <th class="govuk-table__header" scope="col">Prison no.</th>
+      <th class="govuk-table__header" scope="col">Days since sentence</th>
+      <th class="govuk-table__header" scope="col">Date due</th>
+      <th class="govuk-table__header" scope="col">Referred by</th>
+      <th class="govuk-table__header" scope="col"></th>
+    </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+    {% for row in offenders %}
+    <tr class="govuk-table__row ">
+      <td class="govuk-table__cell"><a target="_blank" href="{{ row.offenderNo | offenderLink }}">{{ row.displayName }}</a></td>
+      <td class="govuk-table__cell">{{ row.offenderNo }}</td>
+      <td class="govuk-table__cell">{{ row.daysSinceSentence }}</td>
+      <td class="govuk-table__cell">{{ row.dateRequired }}</td>
+      <td class="govuk-table__cell">{{ row.securityReferredBy or 'Automatic' }}</td>
+      <td class="govuk-table__cell govuk-!-margin-top-2 govuk-!-margin-bottom-8">
+        {% set basicClasses = "tableButton govuk-!-padding-left-3 govuk-!-padding-right-3" %}
+        {% set classes = basicClasses + " grey-button" %}
+        {{ govukButton({
+          text: 'Manage',
```

Comment:
Again not sure if manage is the right word, but happy for other ideas!

Reply:
be good to follow other existing pages, see cancellation or other confirmation pages